application/src/bll/reducers/cartReducer.test.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  cartReducer,
+  addToCart,
+  addAmount,
+  deleteCar,
+  cleanCart,
+  CartPropTypes
+} from './cartReducer';
+import cars from '../../data/database.json';
+
+describe('Cart reducer tests', () => {
+  let initState: Array<CartPropTypes> = [];
+  const defaultCartState: Array<CartPropTypes> = [];
+  let newState: Array<CartPropTypes> = [];
+  beforeEach(() => {
+    initState = cartReducer(defaultCartState, addToCart(cars[0]));
+  });
+  test('add car to cart', () => {
```

Comment:
name of test case should self-explained, like

`should add valid car to cart`

Reply:
i fixed everywhere